torchvision/models/video/_utils.py

```diff
@@ -0,0 +1,81 @@
+import torch.nn as nn
+
+
+__all__ = ["Conv3DSimple", "Conv3DNoTemporal"]
+
+
+class Conv3DSimple(nn.Conv3d):
+    def __init__(self,
+                 in_planes,
+                 out_planes,
+                 midplanes=None,
+                 stride=1,
+                 padding=1):
+
+        super(Conv3DSimple, self).__init__(
+            in_channels=in_planes,
+            out_channels=out_planes,
+            kernel_size=(3, 3, 3),
+            stride=stride,
+            padding=padding,
+            bias=False)
+
+    def forward(self, x):
+        return super().forward(x)
+
+    @staticmethod
+    def get_downsample_stride(stride):
+        return stride
+
+
+class Conv2Plus1D(nn.Module):
```

Comment:
You can also make it inherit from `nn.Sequential`, so that you don't need to add one extra level of nestedness.

```python
conv1 = [
            nn.Conv3d(in_planes, midplanes, kernel_size=(1, 3, 3),
                      stride=(1, stride, stride), padding=(0, padding, padding),
                      bias=False),
            nn.BatchNorm3d(midplanes),
            nn.ReLU(inplace=True),
            nn.Conv3d(midplanes, out_planes, kernel_size=(3, 1, 1),
                      stride=(stride, 1, 1), padding=(padding, 0, 0),
                      bias=False)
    ]
super(Conv2Plus1D, self).__init__(*conv1)
```

Reply:
sounds good